tests/traits/optionsHelpers.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Contains Trait OptionsHelpers.
+ *
+ * @package WP-Auth0
+ *
+ * @since 3.8.1
+ */
+
+/**
+ * Trait Users.
+ */
+trait OptionsHelpers {
+
+	/**
+	 * Instance of WP_Auth0_Options.
+	 *
+	 * @var WP_Auth0_Options
+	 */
+	public static $opts;
+
+	/**
+	 * Set the Auth0 plugin settings.
+	 *
+	 * @param boolean $on - True to turn Auth0 on, false to turn off.
+	 */
+	public static function auth0Ready( $on = true ) {
+		$value = $on ? uniqid() : null;
+		self::$opts->set( 'domain', $value );
```

Comment:
sounds odd, even for a test, that a domain's test value is not a url shaped string but a random string.

Reply:
The value is not used or checked anywhere so it's irrelevant. In this way, I can set this to something to make the plugin "active" and if I ever try to check for a specific value, it will fail (correctly).